spec/controllers/searchgov_urls_controller_spec.rb

```diff
@@ -3,7 +3,6 @@
 describe Admin::SearchgovUrlsController do
   fixtures :users
```

Comment:
whoops - I missed this too! You can remove this line and the line break following.

Reply:
My specs are failing when I comment that line out, unless I transfer that line (`fixtures :users`) into the shared context itself.